config/test.exs

```diff
@@ -5,81 +5,84 @@ config :logger, :console,
   format: "\n$time $metadata[$level] $levelpad$message\n",
   metadata: [:query_time, :response_status]
 
+common_host = [
```

Comment:
With `common_host` being available it might be good to also have a `common_auth` used in the same manner. Going all the way would also mean to use `common_pool` or similar to remove the repetitive pool configuration.